### PR TITLE
Moved colors up to the template from each app css

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: assets/css/griidc.css [hunk 1]
```diff
@@ -30,15 +30,15 @@
     --color-black:#000;
     --color-gradientStart: #022f3a;
     --color-gradientEnd: #4f6e74;
+    --color-transperantGreen: #ddffdd;
```
Comment (mickel1138) [inline assets/css/griidc.css:33]: TranspArant

File: assets/css/griidc-app.css [hunk 2]
```diff
@@ -29,4 +29,21 @@
...
+table.datasets tr:hover td {
+    background-color: var(--color-table-striped);
+}
+
+table.datasets tr td.highlight {
+    background-color:  var(--color-table-striped);
+}
+
+.datasets tr:nth-child(odd) {
+    background-color: #f6f5f4;
```
Comment (mickel1138) [inline assets/css/griidc-app.css:43]: Is this a relevant color to be named?

Comment (praneethpr) [reply assets/css/griidc-app.css:43]: I found only one instance of this

File: public/css/data-discovery-datasets.css [hunk 3]
```diff
@@ -7,14 +7,6 @@ table.datasets td {
     padding: 5px 10px;
 }
 
-table.datasets tr:hover td {
```
Comment (mickel1138) [inline public/css/data-discovery-datasets.css:10]: In this case, I think the CSS should stay here, just change the color name to something like: `--dataset-table-hover-color' and set that in each theme template.

**Comment by mickel1138** [comment]
Maybe it's me, but data discovery is not working (hover)

File: public/css/data-discovery-datasets.css (hunk 3 above)
Comment (mickel1138) [reply public/css/data-discovery-datasets.css:10]: Actually, never mind my second part of the comment, you did that.

File: public/css/dataland-details.css [hunk 4]
```diff
@@ -107,11 +107,6 @@
 	background-color: #DFD;
 }
 
-tr:hover.dsrow
```
Comment (mickel1138) [inline public/css/dataland-details.css:110]: This doesn't even do anything in Dataland, so maybe just move it back, or remove?

Comment (mickel1138) [reply public/css/dataland-details.css:110]: I guess it is removed now.

**Pull request merged**